plans/feat-memory-topic-restructure.md

```
@@ -0,0 +1,165 @@
+# Memory storage: topic-based restructure (PR-A of #1070)
+
+#1029 introduced the atomic "1 fact = 1 file" layout. After PR #1058 +
```

markdownlint-cli2 (0.22.1)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)